extensions/odoo_theme/layout_templates/homepage.html

```diff
@@ -1,124 +1,99 @@
                         </a>
                     </li>
                     <li>
-                        <a href="{{ pathto('applications/productivity/discuss/overview/get_started') }}" class="stretched-link">
-                            {{ _("Get Started with Discuss") }}
+                        <a href="{{ pathto('applications/sales/point_of_sale') }}" class="stretched-link">
+                            {{ _("Point of Sale") }}
                         </a>
                     </li>
                 </ul>
             </div>
             <div class="col-lg-6">
-                <h2 class="border-top pt-4 pt-lg-0"><i class="i-doc-admin d-lg-none me-2"></i>{{ _("Administration") }}</h2>
-                <p class="text-muted">{{ _("Easily setup your first Odoo installation. From downloads to day-to-day database administration, the dedicated tutorials have got you covered.") }}</p>
+                <h2 class="border-top pt-4 pt-lg-0"><a href="{{ pathto('administration') }}">{{ _("Install & Maintain") }}</a></h2>
+                <p class="text-muted">{{ _("Learn how to install, deploy an dupgrade Odoo on premise or on Odoo.sh.") }}</p>
                 <h5 class="text-muted pt-3 text-uppercase fw_semibold">{{ _("Top Links") }}</h5>
                 <ul class="list-unstyled">
                     <li>
-                        <a href=" {{ pathto('administration/install') }} " class="stretched-link">
+                        <a href=" {{ pathto('administration/deployment/install') }} " class="stretched-link">
                             {{ _("Installing Odoo") }}
                         </a>
                     </li>
                     <li>
                         <a href="{{ pathto('administration/deployment/deploy') }}" class="stretched-link">
-                            {{ _("Deploying Odoo") }}
+                            {{ _("Configuring Odoo") }}
                         </a>
                     </li>
                     <li>
-                        <a href="{{ pathto('administration/update') }}" class="stretched-link">
+                        <a href="{{ pathto('administration/maintain/update') }}" class="stretched-link">
                             {{ _("Updating Odoo") }}
                         </a>
                     </li>
                     <li>
                         <a href="{{ pathto('administration/odoo_sh/overview/introduction') }}" class="stretched-link">
-                            {{ _("Introduction to Odoo.sh") }}
+                            {{ _("Odoo.sh: The Odoo Cloud Platform") }}
                         </a>
                     </li>
                 </ul>
             </div>
         </div>
         <div class="row gx-lg-5">
             <div class="col-lg-6">
-                <h2 class="border-top pt-4"><i class="i-doc-dev d-lg-none me-2"></i>{{ _("Developer") }}</h2>
+                <h2 class="border-top pt-4"><a href="{{ pathto('developer') }}">{{ _("Developers") }}</a></h2>
```

Comment:
If this is intended, the title of `developer.rst` should be changed accordingly.

Reply:
ok, I changed s/Developers/Developer